app/controllers/v0/profile/transactions_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module V0
+  module Profile
+    class TransactionsController < ApplicationController
+
+      # before_action { authorize :vet360, :access? }
+      
+      def status
+        transaction = AsyncTransaction::Vet360::Base.refresh_transaction_status(@current_user, vet360_service, transaction_params[:transaction_id])
+        raise Common::Exceptions::RecordNotFound, transaction unless transaction
+        render json: transaction, serializer: AsyncTransaction::BaseSerializer
+      end
+
+      private
+
+      def transaction_params
+        params.permit :transaction_id
+      end
+
+      def vet360_service
```

Comment:
Naming convention in our other Vet360-specific controllers is just `service`. I'd be in favor of sticking with that, since one can pretty easily see that the service is Vet360 by looking at the implementation. Happy to discuss though.